themes/scss/widgets/overlays.scss

```diff
@@ -124,5 +124,50 @@
   }
 }
 
+// loader search
+.CDB-Search {
+  position: absolute;
+  left: 84px;
+  bottom: 16px;
+}
+.CDB-Search-inner {
+  width: 32px;
+  height: 32px;
+  overflow: hidden;
+  transition: width .2s ease-in;
+  &.is-active {
+    width: 184px;
+    .CDB-Search-text {
+      opacity: 1;
+    }
+  }
+}
+.CDB-Search-action {
+  position: relative;
+  width: 32px;
+  cursor: pointer;
+  display: block;
+}
+.CDB-Search-text {
+  position: absolute;
```

Comment:
Properties should be ordered display, position, top, left, width, margin-right, padding, transition, border, border-left, background, font, opacity, vertical-align